modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_disconnected/installing-disconnected-preparations.adoc
+
+[id="installation-adding-registry-pull-secret_{context}"]
+= Adding the registry to your pull secret
+
+Modify your the pull secret for your {product-title} cluster to describe
+your local registry before you install an {product-title} cluster in a
+disconnected environment.
+
+.Prerequisites
+
+* You prepared the bastion host to use in a disconnected installation.
+* You configured a mirror registry to use in your disconnected environment.
+
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Download your `registry.redhat.io` pull secret from the
+link:https://cloud.redhat.com/openshift/install[OpenShift Infrastructure Providers]
+page.
+
+. Generate the base64-encoded user name and password or token for your mirror
+registry:
++
+----
+$ echo -n '<user_name>:<password>' | base64 -w0 <1>
+
+BGVtbYk3ZHAtqXs=
+----
+<1> For `<user_name>` and `<password>`, specify the user name and password that
+you configured for your registry.
+
+. Make a copy of your pull secret:
++
+----
+$ cat ./pull-secret.text | jq .  > <path>/pull-secret-2.json <1>
```

Comment:
if the first was `.text`, the second can just be `pull-secret.json`.  Dropping the `-2` here will also decouple subsequent docs from the number of steps taken when generating this file.